src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs.cljs

```diff
@@ -49,7 +48,7 @@
          #(swap! state dissoc :show-import-overlay?)
          #(swap! state dissoc :show-import-overlay? :server-response)))
       [:div {:style {:float "right" :padding "0 2em 1em 0"}}
-       [comps/Button {:text "Import Configurations..."
+       [comps/Button {:text "Import Configuration from Method Repository..."
```

Comment:
Configurations -> Configuration is definitely clearer. Maybe "from Method Repository" is too verbose. Your call. :+1: